tethysext/atcore/models/file_database/meta_mixin.py

```diff
@@ -0,0 +1,45 @@
+"""
+********************************************************************************
+* Name: file_database.py
+* Author: glarsen
+* Created On: November 2, 2020
+* Copyright: (c) Aquaveo 2020
+********************************************************************************
+"""
+import json
+import logging
+import os
+
+from sqlalchemy import Column
+from sqlalchemy.dialects.postgresql import JSON
+
+log = logging.getLogger('tethys.' + __name__)
+
+
+class MetaMixin(object):
+    meta = Column('metadata', JSON)
+
+    def write_meta(self):
+        """Write a __meta__.json file. Create it if it does not exist."""
+        with open(os.path.join(self.path, '__meta__.json'), 'w') as f:
```

Comment:
You should provide a default implementation for self.path as well. Something like path = '.' maybe?